recipes/pweave/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.25" %}
+{% set sha256 = "1c0f6921196646243eb7ff9eee742305909be2bc7a5eeeb06a7d1f66cc9758c7" %}
+
+package:
+  name: pweave
+  version: {{ version }}
+
+source:
+  fn: Pweave-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pweave/Pweave-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - Pweave = pweave.scripts:weave
+    - pweave = pweave.scripts:weave
+    - Ptangle = pweave.scripts:tangle
+    - ptangle = pweave.scripts:tangle
+    - pypublish = pweave.scripts:publish
+    - pweave-convert = pweave.scripts:convert
```

Comment:
Actually when `setuptools` is in play, I have had a lot of trouble if I don't define these here. Otherwise `setuptools` makes entrypoints that require `pkg_resources`. Adding them here ensures `conda-build` creates the entrypoints so `setuptools` can't mess them up.